torch/fx/passes/graph_drawer.py

```diff
@@ -273,75 +277,60 @@ def _get_str_for_args_kwargs(arg):
-            if not hasattr(tm, "dtype"):
-                print("tm", tm)
-            result += "|" + "dtype" + "=" + str(tm.dtype) + r"\n"
-            result += "|" + "shape" + "=" + str(tuple(tm.shape)) + r"\n"
-            result += "|" + "requires_grad" + "=" + str(tm.requires_grad) + r"\n"
-            result += "|" + "stride" + "=" + str(tm.stride) + r"\n"
+            result = f'''
+            <tr><td>dtype</td><td>{tm.dtype}</td></tr>
+            <tr><td>shape</td><td>{tuple(tm.shape)}</td></tr>
+            <tr><td>requires_grad</td><td>{tm.requires_grad}</td></tr>
+            <tr><td>stride</td><td>{tm.stride}</td></tr>
+            '''
             if tm.is_quantized:
                 assert tm.qparams is not None
                 assert "qscheme" in tm.qparams
                 qscheme = tm.qparams["qscheme"]
-                if qscheme in {
-                        torch.per_tensor_affine,
-                        torch.per_tensor_symmetric,
-                }:
-                    result += "|" + "q_scale" + "=" + str(tm.qparams["scale"]) + r"\n"
-                    result += "|" + "q_zero_point" + "=" + str(tm.qparams["zero_point"]) + r"\n"
-                elif qscheme in {
-                        torch.per_channel_affine,
-                        torch.per_channel_symmetric,
-                        torch.per_channel_affine_float_qparams,
-                }:
-                    result += "|" + "q_per_channel_scale" + "=" + str(tm.qparams["scale"]) + r"\n"
-                    result += "|" + "q_per_channel_zero_point" + "=" + str(tm.qparams["zero_point"]) + r"\n"
-                    result += "|" + "q_per_channel_axis" + "=" + str(tm.qparams["axis"]) + r"\n"
+                if qscheme in {torch.per_tensor_affine, torch.per_tensor_symmetric}:
+                    result += f'''
+                    <tr><td>q_scale</td><td>{tm.qparams["scale"]}</td></tr>
+                    <tr><td>q_zero_point</td><td>{tm.qparams["zero_point"]}</td></tr>
+                    '''
+                elif qscheme in {torch.per_channel_affine, torch.per_channel_symmetric, torch.per_channel_affine_float_qparams}:
+                    result += f'''
+                    <tr><td>q_per_channel_scale</td><td>{tm.qparams["scale"]}</td></tr>
+                    <tr><td>q_per_channel_zero_point</td><td>{tm.qparams["zero_point"]}</td></tr>
+                    <tr><td>q_per_channel_axis</td><td>{tm.qparams["axis"]}</td></tr>
+                    '''
                 else:
                     raise RuntimeError(f"Unsupported qscheme: {qscheme}")
-                result += "|" + "qscheme" + "=" + str(tm.qparams["qscheme"]) + r"\n"
+                result += f'<tr><td>qscheme</td><td>{tm.qparams["qscheme"]}</td></tr>'
             return result
 
         def _get_tensor_label(self, t: torch.Tensor) -> str:
-            return str(t.dtype) + str(list(t.shape)) + r"\n"
+            return f'<tr><td>tensor</td><td>{t.dtype}{list(t.shape)}</td></tr>'
```

Comment:
Don't you want to add `as_html_fragment=True` option?  Or rename it to `_get_html_tensor_label`?